otx/cli/builder/supported_backbone/omz.mmcls.json

```diff
@@ -1,7 +1,7 @@
 {
   "backbones": {
     "mmcls.MMOVBackbone": {
-      "required": ["model_path"],
+      "required": ["model_path_or_model"],
       "options": {
         "model_path": [
```

Comment:
Could you also fix the `model_path` here as well?

Reply:
This has been fixed in https://github.com/openvinotoolkit/training_extensions/pull/1748/commits/29ce2402ad29ffefced8f110add51b82498a41a3. Are you referring something else?

Reply:
What I said is line6. You need to change line6 as well. :)